tests/programs/op_precedence.vi

```diff
@@ -0,0 +1,4 @@
+
+pub fn main(&io: &IO) {
+  io.println("{4 + 4 ** 3 ** 2 / 2}");
```

Comment:
For one-off tests like this I'm often inclined to throw them in the `misc.vi` repl test. Not necessary just throwing it out there.